src/implementations/twig/components/slider/README-tags.md

```diff
@@ -0,0 +1,41 @@
+# ECL Slider
+
+The slider functionality is designed as a standalone feature and does not include a predefined visual template. It can be seamlessly integrated into existing projects.
+Integration is straightforward—simply add `data-ecl-auto-init="Slider"` to a container element with children. The expected markup includes a `<ul>` element with `<li>` children, or a container (e.g., `<div>`) where its children are utilized as individual slides.
+Alternatively a slider instance can be created as usual also manually by:
+
+var elt = document.querySelector('mySelector');
+var slider = new ECL.Slider(elt);
+slider.init();
+
+It comes with two variants, or actually only one, the default is to slide between items like `cards`, all the slides get the same width and a certain number of items will be visible in relation to the available space.
+The variant is `tags` and can be used to slide in a list of `ecl tags`, or even better an `ecl tag set`, this variant would also work with simple links or buttons, elements that gets their width determined by the text that they contain.
```

Comment:
just a remark here: if the variant 'tags' can be used for things like buttons and links, we could use a more generic name for it